org.prevete.Daty.json

```diff
@@ -1,11 +1,5 @@
 {
-    "app-id": "org.prevete.Daty",
-    "runtime" : "org.gnome.Platform",
```

Comment:
You still need the runtime, the runtime-version, the sdk too. it's just the tags that I meant;

Reply:
Sorry, I have not found documentation on tags, so I thought of them of generic tags.

Reply:
Tags are not used here on Flathub at all unless you want to tag the software as proprietary.